tools/coredump/new.go

```diff
@@ -106,6 +106,16 @@ func (tc *trackedCoredump) OpenELF(fileName string) (*pfelf.File, error) {
 	return tc.CoredumpProcess.OpenELF(fileName)
 }
 
+func (tc *trackedCoredump) ExtractAsFile(fileName string) (string, error) {
+	prefixedFileName := tc.prefix + fileName
```

Comment:
Ideally, we should use `path.Join` here and at other places, as `tc.prefix` is a path part.

Reply:
I'd also then fix the whole in follow up PR if ok?